.github/workflows/edr-ci.yml

```diff
@@ -47,11 +47,11 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
 
-      - name: Cargo hack
-        uses: actions-rs/cargo@v1
-        with:
-          command: hack
-          args: check --feature-powerset --no-dev-deps
+      - name: Cargo hack for EDR crates
+        run: |
+          chmod +x ./github/scripts/cargo-hack-edr.sh
+          ./github/scripts/cargo-hack-edr.sh
+        shell: bash
```

Comment:
`cargo-hack`, the tool that makes sure that all feature combinations compile is failing for the Foundry crates. Since we aren't planning to publish them, this is ok, so I changed the CI to only run `cargo-hack` for EDR crates.

Reply:
Update: https://github.com/NomicFoundation/edr/pull/466#issuecomment-2132907024